webv2/updateTableSchema/renameColumn.go

```diff
@@ -0,0 +1,316 @@
+package updateTableSchema
+
+import (
+	"github.com/cloudspannerecosystem/harbourbridge/internal"
+	"github.com/cloudspannerecosystem/harbourbridge/spanner/ddl"
+)
+
+// renameColumn renames given column to newname and update in schema.
+func renameColumn(newName, table, colName string, Conv *internal.Conv) {
+
+	sp := Conv.SpSchema[table]
+
+	renameColumnNameInCurrentTableSchema(Conv, sp, table, colName, newName)
+
+	// update foreignKey relationship Table
+	for i, _ := range sp.Fks {
+
+		renameColumnNameInForeignkeyTableSchema(Conv, sp, i, colName, newName)
+	}
+
+	for _, sp := range Conv.SpSchema {
+
+		for j := 0; j < len(sp.Fks); j++ {
+			if sp.Fks[j].ReferTable == table {
+				renameColumnNameInForeignkeyReferTableSchema(Conv, sp, sp.Name, colName, newName)
+			}
+
+		}
+
+	}
+
+	// update interleave table relation
+	isParent, parentschemaTable := IsParent(table)
+
+	if isParent {
+		renameColumnNameInParentTableSchema(Conv, parentschemaTable, colName, newName)
+
+	}
+
+	childschemaTable := Conv.SpSchema[table].Parent
+
+	if childschemaTable != "" {
+
+		renameColumnNameInchildTableSchema(Conv, childschemaTable, colName, newName)
+
+	}
+
+}
+
+// renameColumnNameInCurrentTableSchema renames given column in Current Table Schema.
+func renameColumnNameInCurrentTableSchema(Conv *internal.Conv, sp ddl.CreateTable, table string, colName string, newName string) {
```

Comment:
Use this method in a generic way instead of creating 5 separate methods

Reply:
Created a common function call renameColumnNameTableSchema